junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -400,6 +401,53 @@ public static Object invokeMethod(Method method, Object target, Object... args)
 		return Optional.empty();
 	}
 
+	/**
+	 * Build the <em>fully qualified name</em> of the passed parameters.
+	 *
+	 * @param clazz the class that declares the method; never {@code null}
+	 * @param methodName the name of the method; never {@code null}
+	 * @param params the parameter types of the method; may be {@code null} or empty
+	 * @return fully qualified method name; never {@code null}
+	 * @see #loadMethod(String)
+	 */
+	public static String getFullyQualifiedMethodName(Class<?> clazz, String methodName, Class<?>... params) {
+		Preconditions.notNull(clazz, "clazz must not be null");
+		Preconditions.notNull(methodName, "methodName must not be null");
+		Preconditions.notNull(params, "params must not be null");
+
+		return String.format("%s#%s(%s)", clazz.getName(), methodName, StringUtils.nullSafeToString(params));
+	}
+
+	/**
+	 * Build the <em>fully qualified name</em> of the passed method.
+	 *
+	 * @param method the method to build the fully qualified name for; never {@code null}
+	 * @return fully qualified method name; never {@code null}
+	 * @see #loadMethod(String)
+	 */
+	public static String getFullyQualifiedMethodName(Method method) {
+		Preconditions.notNull(method, "method must not be null");
+
+		StringBuilder builder = new StringBuilder();
+		builder.append(method.getDeclaringClass().getName());
+		builder.append('#');
+		builder.append(method.getName());
+		if (method.getParameterCount() == 0) {
+			return builder.toString();
+		}
+		Class<?>[] types = method.getParameterTypes();
+		builder.append('(');
+		builder.append(types[0].getName());
+		if (types.length > 1) {
+			for (int i = 1; i < types.length; i++) {
+				builder.append(',');
+				builder.append(types[i].getName());
+			}
+		}
+		builder.append(')');
+		return builder.toString();
+	}
```

Comment:
Can this method just call `getFullyQualifiedMethodName(method.getDeclaringClass(), method.getName(), method.getParameterTypes())`?